src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -26,60 +26,42 @@
     public static final String CLIPPING_RANK_SUM_KEY =              "ClippingRankSum";
     public static final String CULPRIT_KEY =                        "culprit";
-    public static final String SPANNING_DELETIONS_KEY =             "Dels";
     public static final String ORIGINAL_DP_KEY =                    "DP_Orig"; //SelectVariants
-    public static final String DOWNSAMPLED_KEY =                    "DS";
     public static final String EVENT_COUNT_IN_HAPLOTYPE_KEY =       "ECNT"; //M2
     public static final String FISHER_STRAND_KEY =                  "FS";
     public static final String AS_FISHER_STRAND_KEY =               "AS_FS";
     public static final String AS_SB_TABLE_KEY =                    "AS_SB_TABLE";
     public static final String SB_TABLE_KEY =                       "SB_TABLE";
-    public static final String GC_CONTENT_KEY =                     "GC";
     public static final String GQ_MEAN_KEY =                        "GQ_MEAN";
     public static final String GQ_STDEV_KEY =                       "GQ_STDDEV";
     public static final String HAPLOTYPE_SCORE_KEY =                "HaplotypeScore";
     public static final String HI_CONF_DENOVO_KEY =                 "hiConfDeNovo";
-    public static final String HOMOPOLYMER_RUN_KEY =                "HRun";
-    public static final String HARDY_WEINBERG_KEY =                 "HW";
-    public static final String AVG_INTERVAL_DP_KEY =                "IDP"; //DiagnoseTargets
     public static final String INTERVAL_GC_CONTENT_KEY =            "IGC";
     public static final String INBREEDING_COEFFICIENT_KEY =         "InbreedingCoeff";
     public static final String AS_INBREEDING_COEFFICIENT_KEY =      "AS_InbreedingCoeff";
     public static final String EXCESS_HET_KEY =                     "ExcessHet";
     public static final String RAW_GENOTYPE_COUNT_KEY =             "RAW_GT_COUNT";
-    public static final String AS_HETEROZYGOSITY_KEY =              "AS_InbreedingCoeff";
     public static final String LIKELIHOOD_RANK_SUM_KEY =            "LikelihoodRankSum";
     public static final String LO_CONF_DENOVO_KEY =                 "loConfDeNovo";
-    public static final String LOW_MQ_KEY =                         "LowMQ";
     public static final String MLE_ALLELE_COUNT_KEY =               "MLEAC";
     public static final String MLE_ALLELE_FREQUENCY_KEY =           "MLEAF";
-    public static final String MLE_PER_SAMPLE_ALLELE_COUNT_KEY =    "MLPSAC";
-    public static final String MLE_PER_SAMPLE_ALLELE_FRACTION_KEY = "MLPSAF";
     public static final String MAP_QUAL_RANK_SUM_KEY =              "MQRankSum";
     public static final String RAW_MAP_QUAL_RANK_SUM_KEY =          "RAW_MQRankSum";
     public static final String AS_MAP_QUAL_RANK_SUM_KEY =           "AS_MQRankSum";
     public static final String AS_RAW_MAP_QUAL_RANK_SUM_KEY =       "AS_RAW_MQRankSum";
-    public static final String MENDEL_VIOLATION_LR_KEY =            "MVLR";
```

Comment:
Was this from PhaseByTransmission?  Alas, even after the entire duration of Laurent's postdoc, we still never got that code into GATK4.

Reply:
No idea.